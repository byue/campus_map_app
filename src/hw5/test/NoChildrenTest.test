# Tests that a graph can be created, 9 nodes
# can be added to the graph, and that 
# none of these nodes have children.

# Create a graph
CreateGraph graph1

# Add 9 nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6
AddNode graph1 n7
AddNode graph1 n8
AddNode graph1 n9

# List children of disconnected nodes
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3
ListChildren graph1 n4
ListChildren graph1 n5
ListChildren graph1 n6
ListChildren graph1 n7
ListChildren graph1 n8
ListChildren graph1 n9