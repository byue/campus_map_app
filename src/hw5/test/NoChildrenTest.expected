# Tests that a graph can be created, 9 nodes
# can be added to the graph, and that 
# none of these nodes have children.

# Create a graph
created graph graph1

# Add 9 nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1
added node n7 to graph1
added node n8 to graph1
added node n9 to graph1

# List children of disconnected nodes
the children of n1 in graph1 are:
the children of n2 in graph1 are:
the children of n3 in graph1 are:
the children of n4 in graph1 are:
the children of n5 in graph1 are:
the children of n6 in graph1 are:
the children of n7 in graph1 are:
the children of n8 in graph1 are:
the children of n9 in graph1 are: